tests/new_character_test.cpp

```diff
@@ -204,6 +205,9 @@ TEST_CASE( "Generated character with category mutations", "[mutation]" )
 {
     REQUIRE( !trait_TAIL_FLUFFY.obj().category.empty() );
     avatar &u = get_avatar();
+    clear_avatar();
+    u.clear_mutations();
+    u.mutation_category_level.clear();
```

Comment:
This looks like stuff that should be in `clear_avatar`.

Reply:
Sadly it is definitely not because I re-ran the tests locally, adding each line until it stopped complaining.

...But if you mean in general "it should be in there!" I agree, absolutely.

Reply:
Yeah, if moving that in `clear_avatar` breaks too many other tests, that's a thing for a different PR.

Reply:
I didn't actually test moving those lines into clear_avatar, but I'll try to give it a shake after this PR maybe.

Reply:
clear_mutations() is actually already in clear_avatar() (not sure why I thought otherwise) but mutation_category_level.clear() is something I'll try to move in there when my sanity can withstand more test dancing